blog/_posts/2018-07-19-team-sunshine.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Team ☀️ 'Sunshine' on RGSoC 2018!
+layout: post
+created_at: Thu Jul 19 2018
+permalink: blog/2018-07-19-team-sunshine.md
```

Comment:
Hi @sujinleeme ! Thanks for the PR. '.md' is not required when it comes to declaring 'permalink' parameter. Please remove it.

Reply:
Solved #5d7288c